state/writer_test.go

```diff
@@ -168,12 +168,12 @@ func TestSync(t *testing.T) {
          ],
          "mode":"managed",
          "name":"name",
-         "provider":"provider.aws",
+      	 "provider": "provider[\"registry.terraform.io/hashicorp/aws\"]",
```

Comment:
I think the formatting is wrong (or GH)

Reply:
I think it's GH, I don't see anything wrong hehe.

Reply:
![image](https://user-images.githubusercontent.com/17686157/98347140-4f32d480-2017-11eb-9055-2c52ce6f7958.png)
If it's GH then we can resolve this